.dockerignore

```diff
@@ -2,10 +2,10 @@
 **/obj/*
 **/.vs/
 **/.git/
+artifacts/
 tests/
 vsts/
-src/golang.org/
-src/github.com/
-pkg/
-bin/
-artifacts/
\ No newline at end of file
+**/github.com/
```

Comment:
this is going to be under `vendor` anyway, no?

Reply:
This is for the same reason I mentioned in my earlier comment (i.e we install dep using `go get`)